### DISCUSSION TITLE
Understand Survey creation flow modification

### DISCUSSION THREAD

**Discussion by kiranharidas187** [Sunbird-coKreat/Community#10] [Q&A] [answered]
@pallakartheekreddy @Prerna789 @manojSRawat We are currently working on Observation and Survey creation in the Sunbird creation part. While going through the Survey flow we have identified some modifications done in the flow ie when clicking on the Survey tile, it takes us to a different page(PFA). As @Prerna789 and the team were working on these parts, please help us with the following points

- Is it a necessary change or can we revert it?
- What changes have to be made to revert that change?

Attaching the screenshots for refer

![170635296-1e2589ab-11a2-4895-b20c-c55f71aeaf4d](https://user-images.githubusercontent.com/21116904/171026168-a350792a-709d-485e-b96b-1fd02bff3a59.png)

![170635310-e4138c45-c5d4-491d-9f99-49b31d05a466](https://user-images.githubusercontent.com/21116904/171026183-567fa838-e070-4b70-8ef7-fc7e931dbd1b.png)
ence.

**Comment by Prerna789** [comment]
Hi @kiranharidas187 We will be responding to this thread by tomorrow. 
cc @manojSRawat

**Comment by manojSRawat** [comment, marked as answer]
@kiranharidas187 what is the expected behavior for the survey?

Currently, the popup depends on the category definition. 

The 2nd popup is visible in 2 cases:
1. If it's a **question** and the interaction type is not equal to **choice**.
2. if the category definition has more than 1 editor config

You can check the condition [here](https://github.com/Sunbird-Ed/creation-portal/blob/d59018233d2390cad019c3914ef72ecfc30f716d/src/app/client/src/app/modules/shared-feature/components/resource-template/resource-template.component.ts#L96) and the reference function [here](https://github.com/Sunbird-Ed/creation-portal/blob/d59018233d2390cad019c3914ef72ecfc30f716d/src/app/client/src/app/modules/shared-feature/components/resource-template/resource-template.component.ts#L63)

@Prerna789

**Reply by kiranharidas187** [reply]
@manojSRawat User should be taken to question set editor on clicking on Survey tile.

**Reply by kiranharidas187** [reply]
@manojSRawat  After updating schema.properties.mimType.enum to ` [
"application/vnd.sunbird.questionset"
]` it worked fine. Earlier the value was `[
"application/vnd.sunbird.question"
]`

Thanks, @Prerna789 @manojSRawat
